.pre-commit-config.yaml

```diff
@@ -36,7 +36,7 @@ repos:
         exclude: test_p2g.py
 
   - repo: https://github.com/astral-sh/ruff-pre-commit
-    rev: f0fe93c067104b76ffb58852abe79673a8429bd1 # frozen: v0.11.8
+    rev: 73413df07b4ab0bf103ca1ae73c7cec5c0ace593 # frozen: v0.9.2
```

Comment:
Do you want to update these in this PR or a new one?

Reply:
:shrug: I leave it up to @jarrodmillman if he prefers to revert-and-bump in new, separate PRs, or close this and just add a new PR to build on top of this one and undo the unwanted changes in the process.

Reply:
I went ahead and changed this - now the latest versions are preserved and I added the ignoring of the rule that caused all of the autochanges to `pyproject.toml`.